### DISCUSSION TITLE
Go lang installation

### DISCUSSION THREAD

**Discussion by leonjosepulikkottil** [community/community#21511] [Programming Help]
<p>I’ve installed go language package by downloading and extracting</p>
<p>I also changed the path but still its no installed</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Are you following the install instructions from here: <a href="https://golang.org/doc/install" rel="nofollow noopener">https://golang.org/doc/install</a> ?</p>

**Comment by leonjosepulikkottil** [comment]
<p>The issue was due to the path</p>
